src/components/extensive/WizardForm/FormStep.tsx

```diff
@@ -55,7 +60,13 @@ export const FormStep = ({
           uniqueId="name"
           itemClassName="mt-8"
           render={field => (
-            <FieldMapper field={field} formHook={formHook} onChange={onChange} formSubmissionOrg={formSubmissionOrg} />
+            <FieldMapper
+              field={field}
+              formHook={formHook}
+              onChange={onChange}
+              formSubmissionOrg={formSubmissionOrg}
+              projectUuid={projectUuid}
```

Comment:
Drilling down the projectUuid and making `FormStep` and `FieldMapper` more complicated is not great. I'm in the middle of removing this new `formSubmissionOrg` property on my own branch right now. I haven't traced down where this `projectUuid` is coming from (where is it being passed in?), but something that's only needed by one field type should be obtained in that field component another way. A good place to start might be using the [entity context](https://github.com/wri/wri-terramatch-website/blob/staging/src/context/entity.provider.tsx#L18) to get at it. Let me know if you need help with this.